frontend/shoot-client/src/components/Game/Interface3D/SceneFunctions.tsx

```diff
@@ -220,38 +225,59 @@ const buildDeck = (scene: Scene, manager: GUI3DManager) => {
     }
 };
 
-const buildSeatCubes = (scene: Scene, manager: GUI3DManager, camera: ArcRotateCamera) => {
-    var seatCube: SeatCube;
+const buildSeatCubes = (scene: Scene, manager: GUI3DManager, appState: IApp) => {
+    let seatCubes: SeatCube[] = [];
 
     for (var i = 0; i < GameSettings.players; i++) {
         // eslint-disable-next-line
-        seatCube = new SeatCube(scene, manager, camera, i);
+        seatCubes[i] = new SeatCube(scene, manager, i, appState);
     }
+
+    SceneController.seatCubes = seatCubes;
 }
 
 const buildBidCubes = (scene: Scene, manager: GUI3DManager) => {
+    const allBidNumberCubes: BidNumberCube[][] = [];
```

Comment:
don't these being const at least throw a warning?

Reply:
I see what you mean, but they don't seem to -- I think the const seems to only apply to the array pointer itself.